File: algorithms/sorting/bubble-sort/bubble.py
import random

def bubblesort(arr):
    n = len(arr)
    for i in range(n-1):
        for j in range(n-1-i):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]

size = 10
arr = random.sample(range(10,90),size)
     
print('Bubble Sort in Python')
print(f'Original {arr}')
bubblesort(arr)
print(f'Sorted {arr}')

File: algorithms/sorting/bubble-sort/bubble.js
const size = 10
let arr = [];

function input_array(arr){
    for(let i = 0;i < size; i++){
        let num = Math.floor(Math.random() * 90) + 10;
        arr.push(num)
    }
}

function bubblesort(arr){
    let i,j
    for(i = 0;i < size-1;i++){
        for(j = 0;j < (size-1-i);j++){
            if(arr[j] > arr[j+1]){
                [arr[j], arr[j+1]] = [arr[j+1], arr[j]]
            }
        }
    }
}

input_array(arr)
console.log(`Bubble Sort em Javascript\nOriginal : ${arr}`)
bubblesort(arr)
console.log(`Ordenado : ${arr}`)